dashboard/app/assets/stylesheets/application.scss

```diff
@@ -739,25 +739,6 @@ $small-footer-standard-width: 400px;
       background-color: $background-gray;
     }
   }
-
-  .globe-icon {
-    color: $charcoal;
-    font-size: 18px;
-    margin: 0 1px;
-  }
```

Comment:
globe-icon is not used anymore. We have a `::before` pseudoelement containing the globe icon.